src/Illuminate/Validation/Rules/Email.php

```diff
@@ -24,17 +24,17 @@ class Email implements Rule, DataAwareRule, ValidatorAwareRule
 {
     use Conditionable, Macroable;
 
-    public bool $strict = false;
+    public bool $rfcCompliantWithoutWarnings = false;
 
-    public bool $dns = false;
+    public bool $domainExists = false;
 
-    public bool $spoof = false;
+    public bool $preventEmailSpoofing = false;
 
-    public bool $filter = false;
+    public bool $basicFormat = false;
```

Comment:
That could easily confused to be the default, which actually is `rfcCompliant`. So, it might be more intuitive to name it something along the lines of `native(Input|Var|Data)Filter` or the like as it uses PHP's own [Data Filters](https://www.php.net/manual/en/book.filter.php)